libkineto/src/init.cpp

```diff
@@ -136,6 +136,12 @@ bool libkineto_init(bool cpuOnly, bool logOnError) {
   }
 #endif // HAS_CUPTI
 
+#ifdef HAS_ROCTRACER
+  if (!cpuOnly) {
+
```

Comment:
?

Reply:
Oh, that was a placeholder, didn't need anything there.  I'll remove, thanks.